openslides_backend/action/actions/meeting/import_.py

```diff
@@ -570,10 +613,12 @@ def migrate_data(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                     "resource_ids",
                     "organization_tag_ids",
                 ],
+                lock_result=False,
```

Comment:
This must be locked! Everytime relations are fetched, they have to be locked to prevent parallel alterations and therefore lost updates on it. Things that must not be locked are only static fields like `meeting_id` that do not ever change and fields for simple condition checks where the rationale is "if this is changed during this request, we'll just pretend that this request was issued slightly earlier".